components/expenses/ExpenseFormItems.js

```diff
@@ -176,7 +178,8 @@ class ExpenseFormItems extends React.PureComponent {
     }
 
     const onRemove = requireFile || items.length > 1 ? this.remove : null;
-    const availableCurrencies = this.getPossibleCurrencies();
+    const isPayPal = values?.payoutMethod?.type === PayoutMethodType.PAYPAL;
+    const availableCurrencies = isPayPal ? PayPalSupportedCurrencies : this.getPossibleCurrencies();
```

Comment:
:no_entry_sign: This is still bypassing the feature flag check by not calling `getPossibleCurrencies`. The goal of `getPossibleCurrencies` is to return the list of possible currencies, why not do the payout method check there?

Reply:
@Betree : Sorry, think I misunderstood this part yesterday night (😴 😴). I've just corrected this and now I think it should look good.